invenio_search/ext.py

```diff
@@ -269,13 +280,27 @@ def put_templates(self, ignore=None):
         """Yield tuple with registered template and response from client."""
         ignore = ignore or []
 
+        def _replace_prefix(template_path, body):
+            """Replace index prefix in template request body."""
+            pattern = '__SEARCH_INDEX_PREFIX__'
+
+            prefix = self.app.config['SEARCH_INDEX_PREFIX'] or ''
```

Comment:
```suggestion
            prefix = self.app.config.get('SEARCH_INDEX_PREFIX')
```

Reply:
Well, I started like this, but then, I really don't want `None` values, but I want `''` (empty) instead!
Plus, you set it to `None`, `self.app.config.get('SEARCH_INDEX_PREFIX', '')` will not work...